docs/docs/features/partner-sharing.md

```diff
@@ -1,17 +1,62 @@
 # Partner Sharing
 
 Immich allows you to share your library with other users. They can then view your library and download the assets.
-
 You can manage one or multiple users to have access to your library from the [User Settings](docs/features/user-settings.md) page.
 
-<img src={require('./img/partner-sharing-1.png').default} title='Partner Sharing 1' />
+Partner sharing include:
+
+- All your photos and videos except those in Archived and Deleted.
+- You are at the mercy of the partners' actions (as it should be) on the asset they own. If the partner decides to delete or archive an asset belonging to them, it will not be displayed on your timeline.
+- The location where your photos were taken.
+- You can view partners' media and add them to an album. No other actions are allowed on those assets.
+- The search result doesn't include partners' assets (yet)
+- You cannot favorite partners' assets (yet)
+
+:::info
+Partner sharing is one-sided, whoever shared with you will not be able to see your photos  
+unless you share them with them through partner sharing from your user.
+:::
+
+## Add Partner
 
-<img src={require('./img/partner-sharing-2.png').default} title='Partner Sharing 2' />
+:::note Duplicates
+Partner sharing may result in displaying duplicate assets on the main timeline.
+:::
+
+<img src={require('./img/partner-sharing-1.png').default} width="70%" title='Add Partner 1' />
+
+<img src={require('./img/partner-sharing-2.png').default} width="70%" title='Add Partner 2' />
+
+<img src={require('./img/partner-sharing-4.png').default} width="70%" title='Add Partner 4' />
+
+## Accessing to the Shared Library
 
 Accessing the shared library can be done from the Sharing page.
 
-<img src={require('./img/partner-sharing-3.png').default} title='Partner Sharing 3' />
+<img src={require('./img/partner-sharing-3.png').default} width="70%" title='Access to the Shared Library' />
 
 :::tip Sharing specific assets
 For sharing a specific set of assets, you can use the shared album feature of Immich.
 :::
+
+## View partner's photo and video on your timeline
+
+to enable parter-shared photos to be displayed in the main timeline. This feature can be enabled on a per-partner basis and can be viewed and updated on both the web and mobile app.
```

Comment:
Is an image missing here or something? This first part does not make a lot of sense to me

Reply:
Thanks for the review, under this section it is explained how to perform this process.

![image](https://github.com/immich-app/immich/assets/51673860/c7a94606-ae6f-4e2b-846f-7a50345f4e96)